src/storage/sftp/write.c

```diff
@@ -365,10 +365,10 @@ storageWriteSftpClose(THIS_VOID)
 
             if (rc)
             {
-                // some/most sftp servers will not rename over an existing file, in testing this returned LIBSSH2_FX_FAILURE
+                // Some/most sftp servers will not rename over an existing file, in testing this returned LIBSSH2_FX_FAILURE
```

Comment:
also unrelated changes

Reply:
OK. I'll have to remember narrow my commits to related changes.